client/src/components/TabBody/style.css

```diff
@@ -0,0 +1,29 @@
+.tabBody-guidelines {
+  text-align: right;
+  font-size: .5em;
+}
+
+.tabBody-guidelines-href {
```

Comment:
delete all style for the a tag and the hover

Reply:
Why?

Reply:
all this style is written in the style for the app
because it's common style so instead of putting it on each page we put it once in the app style
did you get it?

Reply:
okay